src/libs/PolicyUtils.ts

```diff
@@ -277,6 +277,18 @@ function goBackFromInvalidPolicy() {
     Navigation.navigate(ROUTES.SETTINGS_WORKSPACES);
 }
 
+/** Get a tax with given ID from policy */
+function getTaxByID(policy: OnyxEntry<Policy>, taxID: string): TaxRate | undefined {
+    return policy?.taxRates?.taxes?.[taxID];
+}
+
+/**
+ * Whether the tax rate can be deleted and disabled
+ */
+function canEditTaxRate(policy: Policy, taxID: string): boolean {
+    return policy.taxRates?.defaultExternalID !== taxID;
```

Comment:
Coming from https://github.com/Expensify/App/issues/46657, here we prevent deleting `defaultExternalID` but missed to also prevent `foreignTaxDefault` and it fixed here https://github.com/Expensify/App/pull/46862